edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/ServiceProxy.cs

```diff
@@ -34,23 +33,25 @@ public IServiceIdentitiesIterator GetServiceIdentitiesIterator()
             }
         }
 
-        public async Task<Option<ServiceIdentity>> GetServiceIdentity(string deviceId)
+        public async Task<Option<ServiceIdentity>> GetServiceIdentity(string deviceId, string onBehalfOfDevice)
```

Comment:
check not null or empty for onBehalfOfDevice

Reply:
Async suffix

Reply:
None of the others have the async suffix (they prob should), but updating it means updating the interface and all the related consumers...going to call this one out of scope for this PR =/

---
In reply to: [479393278](https://github.com/Azure/iotedge/pull/3467#discussion_r479393278) [](ancestors = 479393278)